### PR TITLE
[ch6661] Update container label tags

### PR DESCRIPTION
**Pull request by jkatz**
This matches the specified formats that are required by several
distributors, and removes the anti-pattern of having the LABEL subkeys
capitalized.

### CODE REVIEW

File: ubi7/Dockerfile.pgo-backrest.ubi7 [hunk 1]
```diff
@@ -2,8 +2,7 @@ ARG PREFIX
 ARG BASEVER
 FROM ${PREFIX}/pgo-base:ubi7-${BASEVER}
 
-LABEL name="crunchydata/pgo-backrest-" \
-    run='docker run -d -p 8080:80 --name=web-app web-app' \
+LABEL name="crunchydata/pgo-backrest" \
```
Comment (wilybrace) [inline ubi7/Dockerfile.pgo-backrest.ubi7]: Is there a reason this breaks from the naming pattern of the other files?

Comment (jkatz) [reply ubi7/Dockerfile.pgo-backrest.ubi7]: Yes: I missed updating it.

**Comment by crunchyjohn** [comment]
+1

File: rhel7/Dockerfile.pgo-backrest-repo.rhel7 [hunk 2]
```diff
@@ -3,8 +3,7 @@ ARG BASEVER
 FROM ${PREFIX}/pgo-base:rhel7-${BASEVER}
 
 LABEL name="pgo-backrest-repo" \
-	run='docker run -d -p 8080:80 --name=web-app web-app' \
-	summary="Crunchy Data PostgreSQL Operator - pgo-backrest-repo" \
+	summary="Crunchy PostgreSQL Operator - pgo-backrest-repo" \
```
Comment (cbandy) [inline rhel7/Dockerfile.pgo-backrest-repo.rhel7]: I prefer the `summary` and `description` in the CentOS image, `Crunchy PostgreSQL Operator - pgBackRest Repository`.

File: ubi7/Dockerfile.pgo-backrest-repo.ubi7 [hunk 3]
```diff
@@ -3,9 +3,8 @@ ARG BASEVER
 FROM ${PREFIX}/pgo-base:ubi7-${BASEVER}
 
 LABEL name="pgo-backrest-repo" \
-    run='docker run -d -p 8080:80 --name=web-app web-app' \
-	summary="Crunchy Data PostgreSQL Operator - pgo-backrest-repo" \
-	description="Crunchy Data PostgreSQL Operator - pgo-backrest-repo"
+	summary="Crunchy PostgreSQL Operator - pgo-backrest-repo" \
```
Comment (cbandy) [inline ubi7/Dockerfile.pgo-backrest-repo.ubi7]: The `summary` and `description` here differ slightly from the CentOS image.

**Pull request merged**